### ISSUE TITLE
[Bug Report] Icons inside of buttons appear faded out

### ISSUE THREAD

**Issue by Qadosch** [vuetifyjs/vuetify#10466]
**Simillar issue:** [#2108](https://github.com/vuetifyjs/vuetify/issues/2108)

### Environment
**Vuetify Version:** 2.2.9
**Vue Version:** 2.6.10
**Browsers:** Google Chrome
**OS:** Windows 10

### Steps to reproduce
```html
<v-btn
  v-if="customerId === null"
  :loading="loadingExcel"
  :disabled="loadingExcel"
  color="primary"
  @click="downloadExcel"
  small
>
  <v-icon left>mdi-file-excel</v-icon>
  {{ $t("download") }}
</v-btn>
```
adding the dark property to either v-icon or v-btn has no effect
inside vuetify.js
```js
theme: {
  dark: false,
  themes: {
    light: {
      ...
      primary: colors.blue.darken3,
      ...
    },
    dark: {
      ...
      primary: colors.blue.darken3,
      ...
    },
  }
},
```

### Expected Behavior
![grafik](https://user-images.githubusercontent.com/32685630/73853033-7eba7380-4830-11ea-82f1-1062a921b771.png)

### Actual Behavior
![grafik](https://user-images.githubusercontent.com/32685630/73850221-93483d00-482b-11ea-87f2-a4d5301733f6.png)

Inside chrome inspector there is this property
```css
.theme--light.v-icon {
    color: #000000; // inactive
    opacity: 0.54;
}
```

### Reproduction Link
<a href="https://codepen.io/dontknowhowthisshitworksdontevencare" target="_blank">https://codepen.io/dontknowhowthisshitworksdontevencare</a>

**Comment by FryDay** [comment]
I'm also seeing `color: 0.54` in `.theme--light.v-btn.v-btn--icon`. That's not a valid property value.

**Comment by MajesticPotatoe** [comment]
resolved in de967ac
will reflect in next patch
